src/Features/LanguageServer/ProtocolUnitTests/SemanticTokens/SemanticTokensRangeTests.cs

```diff
@@ -118,13 +120,14 @@ public class Template
                 markup, mutatingLspWorkspace, GetCapabilities(isVS));
 
             var document = testLspServer.GetCurrentSolution().Projects.First().Documents.First();
-            var ranges = new LSP.Range[2] {
-                new LSP.Range { Start = new Position(12, 0), End = new Position(13, 0) },
-                new LSP.Range { Start = new Position(29, 0), End = new Position(30, 0) }
-            };
+            ImmutableArray<LinePositionSpan> spans = [
```

Comment:
Hmm, feels like these tests should be calling the actual range handler..., but thats not a new problem in this PR